Tasks/JenkinsQueueJob/jenkinsqueuejobtask.ts

```diff
@@ -111,6 +111,9 @@ async function doWork() {
         var rootJob = await util.pollCreateRootJob(queueUri, jobQueue, taskOptions);
         //start the job queue
         jobQueue.start();
+        //store the job name in the output variable
+        tl.debug('JENKINS_JOB_ID=' + rootJob.executableNumber);
+        tl.setVariable("JENKINS_JOB_ID", rootJob.executableNumber.toString());
```

Comment:
I need to double-check, but I think calling `setVariable()` already provides debug output.